test/controllers/api/v2/environments_controller_test.rb

```diff
@@ -39,6 +39,64 @@ def test_create
       assert_response :success
     end
 
+    def test_create_with_name
+      Organization.any_instance.stubs(:save!).returns(@organization)
+      env_name = 'dev env'
+      assert_difference('KTEnvironment.count') do
+        post :create, params: { :organization_id => @organization.id, :environment => {
+          :name => env_name,
+          :prior => @library.id
+        } }
+      end
+      assert_response :success
+      response = JSON.parse(@response.body)
+      assert response.key?('name')
+      assert_equal response['name'], env_name
+    end
+
+    def test_create_with_description
+      description = 'Environment Description.'
+      assert_difference('KTEnvironment.count') do
+        post :create, params: { :organization_id => @organization.id, :environment => {
+          :name => 'DEV',
+          :description => description,
+          :prior => @library.id
+        } }
+      end
+      assert_response :success
+      response = JSON.parse(@response.body)
+      assert response.key?('description')
+      assert_equal response['description'], description
+    end
```

Comment:
We already have tests in this controller that test creation with a name and description, can you update those?

Reply:
@jlsherrill I wanted to be sure that we are creating the env with the minimum required attributes, but if you think that this will have absolutely no effect, will update that test and remove these one.

Reply:
the first test did that and I'm okay with that (although maybe on the fence).  This one is passing name and description and seems very similar to our other create test.